applications/console_text_messenger/src/main.rs

```diff
@@ -150,7 +150,7 @@ pub fn main() {
 
     let mut settings = Settings::default();
     let mut contacts = Peers { peers: Vec::new() };
-
+    let mut database_path = "./data/text_message_service.sqlite3".to_string();
```

Comment:
This is used more than one, I see below is used as well, just with an added id. 

Perhaps we can use a constant and format where needed, or use a macro taking the id, to format the string.

Reply:
It is only mutated in these two places and the place it is used below this line is a special command line switch used when debugging only. In the long term this will be populated from the global Tari config file which I am not using for these apps yet.